lib/board.rb

```diff
@@ -0,0 +1,70 @@
+module Scrabble
+MAX_LENGTH = 15
+  class Board
+    attr_accessor :board
+
+    def initialize
+      @board = []
+      initialize_board
```

Comment:
Good use of a separate method to encapsulate the `@board` initialization functionality!